ConsistencyManager/ConsistencyManager.swift

```diff
@@ -755,6 +755,10 @@ public class ConsistencyManager {
                 // Default to true if weakOperation is nil
                 return weakOperation?.cancelled ?? true
             }
+            // Let's check here to see if it's cancelled before we start
+            if cancelled() {
```

Comment:
super-minor: instead of `cancelled()` here, I'd put `()` at the end of the `{ }` block above, so you can just say `if cancelled`